javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/ATMConfig.qll

```diff
@@ -127,6 +141,17 @@ abstract class AtmConfig extends string {
    * A cut-off value of 1 produces all alerts including those that are likely false-positives.
    */
   float getScoreCutoff() { result = 0.0 }
+
+  /**
+   * Holds if there's an ATM alert (a flow path from `source` to `sink` with ML-determined likelihood `score`) according
+   * to this ML-boosted configuration, whereas the unboosted base query does not contain this source and sink
+   * combination.
+   */
+  predicate getAlerts(JS::DataFlow::PathNode source, JS::DataFlow::PathNode sink, float score) {
```

Comment:
Minor/bikeshed: I'd prefer a name that doesn't suggest a return/result value, and mentions flow. Also makes it easier to see the parallels between unboosted and boosted queries. How about `hasLikelyFlowPath`?

Reply:
I don't know about `hasLikelyFlowPath`, because one of the conditions it _must_ meet is `hasFlowPath`. Logically `hasFlowPath` should imply `hasLikelyFlowPath`. 

Maybe something like `hasBoostedFlowPath`? We've been trying to get rid of the "boosted" terminology, but I can't think of something better right now. I think `hasAlert` would be clearer (ultimately that's what this predicate does -- it surfaces all the alerts for this config), but that doesn't mention flow.

Reply:
Note: I tracked this renaming together with the other renamings [here](https://github.com/github/ml-ql-adaptive-threat-modeling/issues/2136), so that we can merge this PR and XssThroughDOM and open a separate PR for all the renamings